build.sbt

```diff
@@ -8,9 +7,12 @@ val codeguruURI =
 // the application
 lazy val hat = project
   .in(file("hat"))
+  .settings(dockerSettings)
   .enablePlugins(PlayScala)
   .enablePlugins(SbtWeb, SbtSassify, SbtGzip, SbtDigest)
   .enablePlugins(BasicSettings)
+  .enablePlugins(SlickCodeGeneratorPlugin)
+  .disablePlugins(PlayLogback)
```

Comment:
Isn't this play logs? Are we using something else to log stuff?

Reply:
this (plus the removal of the no-op logger) makes slf4j work correctly
and log stacktraces for example